compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1111,13 +1111,21 @@ rustc_queries! {
         desc { "getting a &core::panic::Location referring to a span" }
     }
 
-    // FIXME get rid of this with valtrees
     query lit_to_const(
         key: LitToConstInput<'tcx>
-    ) -> Result<ty::Const<'tcx>, LitToConstError> {
+    ) -> Option<ty::Const<'tcx>> {
```

Comment:
this query could alternatively *just* return an (option of) valtree to make it clear what it's doing